client/src/utils/storage.js

```diff
@@ -0,0 +1,7 @@
+export const saveLoggedInUserData = data =>
+  localStorage.setItem("userData", JSON.stringify(data));
+
+export const getLoggedInUserData = () =>
+  JSON.parse(localStorage.getItem("userData"));
+
+export const removeUserDataOnLogout = () => localStorage.removeItem("userData");
```

Comment:
`onLogout` is also a bad name ... this module doesn't care when the removing happens, it's not its concern - it can be on logout, on clearing cache, maybe there is switching users .. it's irrelevant for `storage` module to know about the event triggering it. Just name it `removeUserData`.

Reply:
Done